ui/src/dashboards/containers/DashboardPage.js

```diff
@@ -68,8 +70,11 @@ class DashboardPage extends Component {
       isUsingAuth,
       router,
       notify,
+      getAnnotationsAsync,
     } = this.props
 
+    const fifteenMinutes = Date.now() - 15 * 60 * 1000
+    getAnnotationsAsync(source.links.annotations, fifteenMinutes)
```

Comment:
getAnnotationsAsync only ever gets called for the past fifteen minutes, should be dashboard time interval
.

Reply:
:+1: